src/Model/StudentManager.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Model;
+
+class StudentManager extends AbstractManager
+{
+    public const TABLE = 'Student_Reviews';
```

Comment:
since it is the student manager, start with the table student for simpler use

Reply:
I try to fetch Student table first; then fectch Student reveiw table. It seem like this